Assignment 3

chevrie4 opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Adding two md files, two png files, one jupyter notebook.

## What did you learn from the changes you have made?
How to use Seaborn to make visualizations

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
Variety of other Seaborn plots and also datasets to choose from.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Difficult to replicate Seaborn plot on Excel.

## How were these changes tested?
Reviewing the dataset manually to ensure visualizations look accurate.

## A reference to a related issue in your repository (if applicable)

## Checklist
- [X] I can confirm that my changes are working as intended

chevrie4 committed:
submitting visualization assignment 1

chevrie4 committed:
submitting assignment 1 week 2 code

chevrie4 committed:
Submitting Assignment 3

chevrie4 committed:
submitting assignment 3

xindizhang reviewed (changes requested):
Great work! I really appreciate your organized submission. Your visualizations are clear, informative, and demonstrate a solid understanding of the class content. The topic you chose is very meaningful, and I especially loved the low cognitive load in your visuals—well done!

Your code is reproducible—no changes requested! Just a quick note:
- you have an error in your fourth-to-last code block `NameError: name 'ax' is not defined`
- and third-to-last chunk of code: `NameError: name 'df' is not defined`

**Mark: Change Requested**
- To complete the assignment, please include one visualization created using a different data visualization tool (other than Python), as required. Could you please confirm if you’ve already submitted a visualization using another tool?

Feel free to let me know if you have any questions!

chevrie4 committed:
Delete 02_activities/assignments/assignment_3 directory

Deleting separate file for assignment 3 to clean up the assignment directory

chevrie4 committed:
updating description for viz 2 and ipynb

chevrie4 commented:
> Great work! I really appreciate your organized submission. Your visualizations are clear, informative, and demonstrate a solid understanding of the class content. The topic you chose is very meaningful, and I especially loved the low cognitive load in your visuals—well done!
> 
> Your code is reproducible—no changes requested! Just a quick note:
> 
> * you have an error in your fourth-to-last code block `NameError: name 'ax' is not defined`
> * and third-to-last chunk of code: `NameError: name 'df' is not defined`
> 
> **Mark: Change Requested**
> 
> * To complete the assignment, please include one visualization created using a different data visualization tool (other than Python), as required. Could you please confirm if you’ve already submitted a visualization using another tool?
> 
> Feel free to let me know if you have any questions!

Thanks very much for reviewing my assignment!  I have updated the a3-v2 markdown file to specify that the second viz (a3-v2.png) was indeed created using a data visualization tool other than Python (MS Excel) and updated the notebook to remove the errors you described.
